attributes/default.rb

```diff
@@ -23,6 +23,12 @@
 #
 default['chef-ingredient']['mixlib-install']['git_ref'] = nil
 
+#
+# Optionally override the omnitruck endpoint
+#
+default['chef-ingredient']['omnitruck']['endoint'] = nil
```

Comment:
Can we document omnitruck and mixlib-install-git-ref properties as development / testing only? 

I do not expect people to change omnitruck and they should be aware that git will be installed onto their system when they set git-ref for mixlib-install.